design/index.md

```diff
@@ -141,3 +141,7 @@ Keep it short, Reddit posts should be factual, informal and have a call to actio
 #### Node-RED Slack (FlowForge section)
 
 Keep it short, Slack messages should be factual, informal and have a call to action (a link) where a user can read more.
+
+#### Discord (Node-RED Forums)
```

Comment:
@robmarcer Discord is a chat server, I think you mean `discourse`?

Reply:
I do indeed @ZJvandeWeg, I've updated it to say Discourse, can you review again please?